renderdoc/driver/gl/wgl_platform.cpp

```diff
@@ -59,14 +59,18 @@ class WGLPlatform : public GLPlatform
     };
 
     ret.ctx = WGL.wglCreateContextAttribsARB(share.DC, share.ctx, attribs);
+    ret.DC = GetDC(ret.wnd);
```

Comment:
As I mentioned, `GetDC` may not always be safe and I don't think we're necessarily guaranteed to have a valid window here. Unless it doesn't work, `CreateCompatibleDC` should be safer as that creates an actual new DC instead of getting a new handle to an existing DC.

Reply:
Yeah this didn't work. I have updated the PR with a solution that addresses your concern, although it's still very awkward. Let me know if you have a better solution to try!